.editorconfig

```diff
@@ -0,0 +1,9 @@
+
+# top-most EditorConfig file
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = tab
```

Comment:
please help could we (=more than one) clarify if that is confusing existing files? also asking: @bufferoverflow @steffen-weber